cl/antiquary/tests/tests.go

```diff
@@ -128,6 +140,29 @@ func LoadChain(blocks []*cltypes.SignedBeaconBlock, s *state.CachingBeaconState,
 	return m
 }
 
+func GetElectraRandom() ([]*cltypes.SignedBeaconBlock, *state.CachingBeaconState, *state.CachingBeaconState) {
+	block1 := cltypes.NewSignedBeaconBlock(&clparams.MainnetBeaconConfig, clparams.ElectraVersion)
+	block2 := cltypes.NewSignedBeaconBlock(&clparams.MainnetBeaconConfig, clparams.ElectraVersion)
+
+	// Lets do te
+	if err := utils.DecodeSSZSnappy(block1, electra_blocks_0_ssz_snappy, int(clparams.ElectraVersion)); err != nil {
+		panic(err)
+	}
+	if err := utils.DecodeSSZSnappy(block2, electra_blocks_1_ssz_snappy, int(clparams.ElectraVersion)); err != nil {
+		panic(err)
+	}
+	preState := state.New(&clparams.MainnetBeaconConfig)
+	if err := utils.DecodeSSZSnappy(preState, electra_pre_state_ssz_snappy, int(clparams.DenebVersion)); err != nil {
+		panic(err)
+
+	}
+	postState := state.New(&clparams.MainnetBeaconConfig)
+	if err := utils.DecodeSSZSnappy(postState, electra_post_state_ssz_snappy, int(clparams.DenebVersion)); err != nil {
```

Comment:
ahhh, I think here should be Electra, because version upgrade is expected in this testcase